benchmark.js

```diff
@@ -0,0 +1,27 @@
+const Web3 = require('web3');
+const mineGasForTransaction = require('./skale-miner').mineGasForTransaction
+
+// Variables definition
+const addressFrom = '0x6Be02d1d3665660d22FF9624b7BE0551ee1Ac91b';
+const addressTo = '0xB90168C8CBcd351D069ffFdA7B71cd846924d551';
+const web3 = new Web3('https://data-seed-prebsc-1-s1.binance.org:8545/');
```

Comment:
Is this some specific endpoint or may we replace web3 by web3-utils for `toWei` and `toHex` functions?

Reply:
These are randomly generated addresses, just for testing.